### DISCUSSION TITLE
failed to send notification for Kustomization/flux-system/flux: failed to extract commit hash

### DISCUSSION THREAD

**Discussion by rwlove** [fluxcd/flux2#4675] [Q&A] [answered]
```flux-system   0s          Warning   NotificationDispatchFailed   alert/github                                             failed to send notification for Kustomization/flux-system/flux: failed to extract commit hash from 'v2.2.3@sha256:187f6a409354d1a6156bb9ede0c81da13e1daccef421203575d23679ccc9b320' revision```

I'm getting multiple events with the above warning. 

I have a PAT for github that has `Read access to metadata` and `Read and Write access to commit statuses`.

I'm not sure why I'm getting this warning. Any ideas?

**Comment by rwlove** [comment]
bump. any ideas?

**Reply by stefanprodan** [reply, marked as answer]
Commit status only works for repos where you've run `flux bootstrap` and for Flux Kustomizations with source GitRepository. In your case, you are using an OCIRepository source, Git has no SHA256 digests.

**Reply by d3nnxs** [reply]
Hi,
I think it would be very cool to make this work also for OCI sources which are based on a git repository. In our case we set the `--revision` flag on the `flux push artifact` command which points to the current revision where the oci where build from. I can also see that this information is saved within the oci source in `status.metadata["org.opencontainers.image.revision"]` .

I do not know the source code very well but if you (@stefanprodan) think it is possible to implement a logic like `if kustomize source is oci than take the revision from the status.metadata` I can try to implement it.
